frappe/__init__.py

```diff
@@ -274,20 +274,14 @@ def init(site: str, sites_path: str = ".", new_site: bool = False, force=False)
 		local.conf.db_user = local.conf.db_name
 
 
-def connect(
```

Comment:
Better not to break existing usage of `connect()` - can leave the method as-is and leave the rest of the changes? (Calling init on our own)

Reply:
Good idea, done in 4e601af915